util.py

```diff
@@ -85,12 +85,12 @@ def create_mom_output(ocean_grid, filename, start_date, history):
 
     f = nc.Dataset(filename, 'w')
 
-    f.createDimension('GRID_X_T', ocean_grid.num_lon_points)
-    f.createDimension('GRID_Y_T', ocean_grid.num_lat_points)
-    f.createDimension('ZT', ocean_grid.num_levels)
+    f.createDimension('lon', ocean_grid.num_lon_points)
+    f.createDimension('lat', ocean_grid.num_lat_points)
+    f.createDimension('depth', ocean_grid.num_levels)
     f.createDimension('time')
```

Comment:
```suggestion
    f.createDimension('time', 1)
```

Reply:
Changed in this commit [95da76f](https://github.com/COSIMA/ocean-regrid/pull/5/commits/95da76f1c042bd9c412d1c50416d1ec9fd21adaa)